src/patient-search/patient-search.component.css

```diff
@@ -0,0 +1,126 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+/** Desktop **/
+@media screen and (min-width: 769px) {
+  .patientSearchHeader {
+    background-color: var(--omrs-color-bg-high-contrast);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .searchResults {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .searchHelper {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .resultsCount {
+    margin-left: 3rem;
+  }
+
+  .patientSearchInput {
+    width: 40%;
+  }
+
+  .helperText {
+    width: 40%;
+  }
+}
+
+/** Tablet/Mobile **/
+@media screen and (max-width: 769px) {
+  .patientSearchHeader {
+    background-color: var(--omrs-color-bg-medium-contrast);
+  }
+}
+
+.resultsCount {
+  font-size: 17px;
+  line-height: 147.6%;
+  padding: 0.5rem 0 0 1.5rem;
+}
+
+.patientSearchHeader {
+  display: flex;
+  align-items: center;
+  height: var(--omrs-topnav-height);
+  box-shadow: inset 0px -2px 0px rgba(0, 0, 0, 0.08);
+  backdrop-filter: blur(24px);
+}
+
+.patientSearchInput {
+  background: transparent;
+  border: none;
+  caret-color: var(--omrs-color-interaction);
+  padding: 0;
+}
+
+.patientSearchInput:focus {
+  outline: none;
+}
+
+.patientSearchInput::placeholder {
+  color: var(--omrs-color-ink-medium-contrast);
+}
+
+.iconBtn {
+  border: none;
+  background: none;
+  cursor: pointer;
+}
+
+.iconBtn:focus {
+  outline: none;
+}
+
+.helperText {
+  color: var(--omrs-color-ink-medium-contrast);
+  margin-left: 2.375rem;
+}
+
+.searchResults {
+  background-color: var(--omrs-color-bg-lowest-contrast);
+}
+
+.resultsText {
+  color: var(--omrs-color-ink-medium-contrast);
+}
+
+.emptyResultContainer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 4rem;
+}
+
+.emptyResultCard {
+  border: 1px solid var(--omrs-color-ink-lowest-contrast);
+  border-radius: 0.5rem;
+  padding: 1.5rem;
+}
+
+.emptyResultText {
+  margin-top: 1rem;
+}
+
+ul.dash {
+  list-style: none;
+  margin-left: 0;
+  padding-left: 1em;
```

Comment:
hmm any reason we're using em instead of rem here?

Reply:
fixed